server/src/main/java/org/cloudfoundry/identity/uaa/zone/ZoneAwareClientSecretPolicyValidator.java

```diff
@@ -82,8 +82,7 @@ public void validate(String clientSecret) throws InvalidClientSecretException {
             clientSecretPolicy = zone.getConfig().getClientSecretPolicy();
         }
 
-        PasswordValidator clientSecretValidator = validator(clientSecretPolicy,
```

Comment:
no, the issue is seperate.

As mentioned before use an older UAA start it and run uaac to create an client with empty secret......

Reply:
I don't want to reproduce the issue. I'm trying to find out what new code is the source of the problem. It seems to me like it should be documented in the fix, so that people understand what is going on.